stdlib/sql/src/main/ballerina/src/sql/types.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/config;
+import ballerinax/java;
+
+# Represents the properties which are used to configure DB connection pool.
+# Default values of the fields can be set through the configuration API.
+#
+# + maxOpenConnections - The maximum number of open connections that the pool is allowed to have, including both idle and in-use connections.
+#                     Default value is 15 and it can be changed through the configuration API with the key
+#                     `b7a.sql.pool.maximumPoolSize`.
+# + minIdleConnections - The minimum number of idle connections that pool tries to maintain in the pool. Default is the same as
+#                 maxOpenConnections and it can be changed through the configuration API with the key
+#                 `b7a.sql.pool.minimumIdle`.
+# + maxConnectionLifeTimeSeconds - The maximum lifetime of a connection in the pool. Default value is 1800 seconds (30 minutes)
+#                  and it can be changed through the configuration API with the key `b7a.sql.pool.maxLifetimeInMillis`.
+#                  A value of 0 indicates unlimited maximum lifetime (infinite lifetime).
+public type ConnectionPool record {|
+    int maxOpenConnections = config:getAsInt("\"b7a.sql.pool.maxOpenConnections\"", 15);
+    decimal maxConnectionLifeTimeSeconds = <decimal>config:getAsFloat("b7a.sql.pool.maxConnectionLifeTime", 1800.0);
```

Comment:
```suggestion
    decimal maxConnectionLifeTimeInSeconds = <decimal>config:getAsFloat("b7a.sql.pool.maxConnectionLifeTimeInSeconds", 1800.0);
```

Reply:
Shall we make sure the property name and config key is same, so it is easy to map